overlay.nix

```diff
@@ -64,6 +59,8 @@ in final: prev: {
   haproxy-cors = prev.callPackage ./pkgs/haproxy-cors.nix { };
 
   devShell = final.callPackage ./pkgs/dev-shell.nix { };
+  bitteShell = final.callPackage ./pkgs/bitte-shell.nix { };
+  bitteSchoell = final.bitteShell;
```

Comment:
What's the point of the redundant shell package?

Reply:
If you mean shell/schoell then that was just for my own lols. Bitte schön, shell, schöll. Meant to remove it but got distracted by popsicles.
If you mean devShell/bitteShell then devShell is this repo's devShell and bitteShell is a devShell template for consumer flakes (`*-ops`es).